composer.json

```diff
@@ -22,7 +22,9 @@
         "json-ld/tests": "@dev"
     },
     "autoload": {
-        "psr-0": { "ML\\JsonLD": "" }
+        "psr-4": { "ML\\JsonLD\\": "src/" }
```

Comment:
Without it, on a production server the Tests are also autoloaded. Because the tests folder is also included in the primary source. 
This is quite a bit of overhead. I always use an `autoload` for the source, and an `autoload-dev` for the tests. That is the reason for the `src` folder being separated from the `tests` folder.

I'm not really sure why the tests fail. But let me check that out.